addon/services/theme.js

```diff
@@ -0,0 +1,30 @@
+import Ember from 'ember';
```

Comment:
Note for line above: do we really need to add this logo twice to the same repo?

`share-logo-icon.png`